Documentation/ceph-quickstart.md

```diff
@@ -96,6 +96,55 @@ rook-ceph-osd-1-5b96b56684-r7zsp       1/1       Running     0          1m
 rook-ceph-osd-prepare-mynode-ftt57     0/1       Completed   0          1m
 ```
 
+# Support for authenticated docker registries
```

Comment:
This topic would apply to all storage providers (ceph, minio, cockroachdb, nfs, etc). Making it more generic and putting it in the prerequisites section as its own page sounds good to me as well.